### ISSUE TITLE
Broken down on Node.js version 9.x and below

### ISSUE THREAD

**Issue by MuffinK** [axelrindle/github-version-checker#7]
**Describe the bug**
Broken down on Node.js version 9.x and below.
```
TypeError: "listener" argument must be a function
    at ClientRequest.once (events.js:340:11)
    at new ClientRequest (_http_client.js:174:10)
    at Object.request (http.js:39:10)
    at Object.request (https.js:245:15)
    at Object.get (https.js:249:21)
    at rest (/usr/lib/node_modules/homebridge-broadlink-rm-pro/node_modules/github-version-checker/lib/check.js:121:19)
    at module.exports (/usr/lib/node_modules/homebridge-broadlink-rm-pro/node_modules/github-version-checker/lib/check.js:164:5)
    at module.exports (/usr/lib/node_modules/homebridge-broadlink-rm-pro/node_modules/github-version-checker/lib/main.js:14:5)
    at checkForUpdates (/usr/lib/node_modules/homebridge-broadlink-rm-pro/test.js:12:3)
    at Object.<anonymous> (/usr/lib/node_modules/homebridge-broadlink-rm-pro/test.js:19:1)
```

**To Reproduce**
use Node.js 9.x

**Expected behavior**
don't throw error

**Versions**
| Component | Version |
| ---- | ---- |
| Node.JS | 8.6.0 |
| github-version-checker | 2.2.0 |

**Additional context**

As discribed in the  [docs](https://nodejs.org/docs/latest-v9.x/api/https.html#https_https_get_options_callback), Node.js v9.x don't support 
```javascript
https.get(url[, options][, callback])
```
https://github.com/axelrindle/github-version-checker/blob/32e01779029bff881b27034bf1519650e497aabd/lib/check.js#L119

**Comment by MuffinK** [comment]
#8

**Comment by axelrindle** [comment]
Closed in #8
